pallets/randomness/src/lib.rs

```diff
@@ -222,7 +222,7 @@ pub mod pallet {
 	/// Removed once $value.request_count == 0
 	#[pallet::storage]
 	#[pallet::getter(fn randomness_results)]
-	pub(crate) type RandomnessResults<T: Config> =
+	pub type RandomnessResults<T: Config> =
 		StorageMap<_, Twox64Concat, RequestType<T>, RandomnessResult<T::Hash>>;
```

Comment:
Is this a breaking change? I needed to make this change to mock fulfillment for the fulfill_request succeeds branch

Reply:
There is already a getter (line 224), so it's not a breaking change from RPC point of view